bigquery/cloud-client/src/test/java/com/example/bigquery/RunDeleteDatasetIT.java

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright 2019 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.example.bigquery;
+
+import static org.junit.Assert.assertFalse;
+
+import com.google.cloud.bigquery.BigQuery;
+import com.google.cloud.bigquery.BigQueryOptions;
+import com.google.cloud.bigquery.DatasetId;
+import com.google.cloud.bigquery.testing.RemoteBigQueryHelper;
+import org.junit.Test;
+
+public class RunDeleteDatasetIT {
+
+  @Test
+  public void deleteDataset() {
+    BigQuery bigquery = BigQueryOptions.getDefaultInstance().getService();
```

Comment:
In order to test without returning a value, just capture the output of the snippet and verify it's printing the correct data. For a good example of how to do this, [see this test](https://github.com/GoogleCloudPlatform/java-docs-samples/blob/master/dlp/src/test/java/dlp/snippets/InspectTests.java#L65).